[HCCDOC#1765]: Add recommendations-filtering content to Remote Health Monitoring (#70364)
openshift/openshift-docs: modules/insights-advisor-recommendations-filters.adoc

// Module included in the following assemblies:
//
// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc

:_mod-docs-content-type: CONCEPT
[id="insights-operator-advisor-recommendation-filters_{context}"]
= Advisor recommendation filters

The Insights advisor service can return a large number of recommendations. To focus on your most critical recommendations, you can apply filters to the https://console.redhat.com/openshift/insights/advisor/recommendations[Advisor recommendations] list to remove low-priority recommendations.

By default, filters are set to only show enabled recommendations that are impacting one or more clusters. To view all or disabled recommendations in the Insights library, you can customize the filters.

To apply a filter, select a filter type and then set its value based on the options that are available in the drop-down list. You can apply multiple filters to the list of recommendations.

You can set the following filter types:

> 

Reviewer: Not sure about this, but maybe the items here would be easier to read at a glance if they were in a table rather than a bulleted list? WDYT?

In any case, per [the ISG guidance on the wording of list items](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#wording-of-list-items), all of the first sentences of the descriptions must be made grammatically parallel. Re-write so that either they all begin with an imperative "Search for...", "Filter by.." to form complete sentences or they begin with a descriptive sentence fragment (as in the "Total risk" item).

Author: done

* *Name:* Search for a recommendation by name.

> * *Total risk:* An abstract value derived from the likelihood that the detected issue will negatively affect your infrastructure, and the impact on operations if that were to happen. Select one or more values from *Critical*, *Important*, *Moderate*, and *Low*.

Reviewer: ```suggestion
* *Total risk:* An abstract value derived from the likelihood that the detected issue will negatively affect your infrastructure and the potential impact on operations. Select one or more values from *Critical*, *Important*, *Moderate*, and *Low*.
```
- remove unneeded comma before subordinating conjunction
- edit to reduce extra words.

> * *Impact:* The potential impact to the continuity of cluster operations. Select from *Critical*, *High*, *Medium*, and *Low*.

Reviewer: ```suggestion
* *Impact:* The potential impact to the continuity of cluster operations. Select one or more values from *Critical*, *High*, *Medium*, and *Low*.
```
- Not sure if this change is accurate, but the previous list item used this construction.

Author: Good change. The meaning isn't changed. Done.

> * *Likelihood:* Filter by the potential for a negative impact to clusters if the recommendation comes to fruition. *Critical*, *High*, *Medium*, or *Low*.

Reviewer: ```suggestion
* *Likelihood:* Filter by the potential for a negative impact to clusters if the recommendation comes to fruition. Select one or more values from *Critical*, *High*, *Medium*, or *Low*.
```
- Same comment as for the previous item. In general, try to refer to these options in a consistent way for each item in the list.

Author: Done

> * *Category:* Select one or more categories of recommendations to concentrate on: *Service Availability*, *Performance*, *Fault Tolerance*, *Security*, *Best Practice*.

Reviewer: ```suggestion
* *Category:* Filter by categories of recommendations. Select one or more values from *Service Availability*, *Performance*, *Fault Tolerance*, *Security*, and *Best Practice*.
```
Same comment as previously. In general, try to refer to these options in a consistent way for each item in the list.

> * *Status:* Set to only show *Enabled* recommendations by default, you can set the filter to show disabled recommendations.

Reviewer: ```suggestion
* *Status:* Set to only show *Enabled* recommendations by default. You can also set the filter to show disabled recommendations.
```
- The meaning of this sentence is not clear to me. Why include the second part of the sentence? I don't know the functionality, so I'm not sure what I'm missing here. Is the setting for showing disabled recommendations something extra?

> * *Clusters impacted:* Also set by default, you can filter to only recommendations currently impacting one or more clusters, and or show non-impacting recommendations.

Reviewer: ```suggestion
* *Clusters impacted: Filter only those recommendations currently impacting one or more clusters. You can also show non-impacting recommendations. This setting is enabled by default.
```
- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-slashes#general-text), do not use the construction "and/or".
